<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f5f5f5;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>256</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>☰</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▐▐</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▶</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>◼</string>
     </property>
    </widget>
    <widget class="QToolButton" name="batteryIndicator">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>330</y>
       <width>29</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>⚡</string>
     </property>
    </widget>
    <widget class="QToolButton" name="onoffButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>29</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>I/O</string>
     </property>
    </widget>
    <widget class="QLabel" name="contactLight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #20a7db;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="treatmentLight">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #4ca64c;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="contactLostLight">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff4c4c;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="headsetPanel">
     <attribute name="title">
      <string>EEG Headset</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="adminPanel">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNeureset">
    <property name="title">
     <string>Neureset</string>
    </property>
   </widget>
   <addaction name="menuNeureset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
